.github/common_environment.yml

```diff
@@ -0,0 +1,9 @@
+  DOCKERHUB_REPOSITORY:  dfedigital/get-into-teaching-api
+  DOMAIN:                london.cloudapps.digital
+  APPLICATION:           Get Into Teaching API Service
+  PAAS_APPLICATION_NAME: get-into-teaching-api
+  SLACK_FAILURE:         '#ff0000'
+  SLACK_SUCCESS:         '#00ff00'
+  SLACK_ICON:            https://raw.githubusercontent.com//DFE-Digital/get-into-teaching-api/master/.github/image.png?size=48
+  SLACK_USERNAME:        GiT Workflows
+  SLACK_FOOTER:          Get Into Teaching API Service
```

Comment:
Would it make sense (and is it possible) - to use `${{ env.APPLICATION }}` here - it seems to duplicate that value, means if the app name gets changed, the footer is automatically updated?

Reply:
it would but I don't think our code is advanced enough just yet